wolfcrypt/src/port/nxp/ksdk_port.c

```diff
@@ -524,6 +538,79 @@ int mp_exptmod(mp_int *G, mp_int *X, mp_int *P, mp_int *Y)
     return res;
 }
 
+int mp_exptmod_nct (mp_int * G, mp_int * X, mp_int * P, mp_int * Y)
+{
+    /* use hardware implementation even for non-constant time operations */
+    return mp_exptmod(G, X, P, Y);
+}
+
+#if !defined(NO_DH) || !defined(NO_DSA) || !defined(NO_RSA) || \
+    defined(WOLFSSL_KEY_GEN)
+int mp_prime_is_prime_ex(mp_int* a, int t, int* result, WC_RNG* rng)
+{
+    int res = MP_OKAY;
+    int szA;
+
+    szA = mp_unsigned_bin_size(a);
+    if (szA <= LTC_MAX_INT_BYTES) {
+        uint16_t sizeA, sizeB;
+        uint8_t *ptrA, *ptrB;
+
+        sizeB = mp_count_bits(a);
+        /* The base size is the number of bits / 8. One is added if the number
+         * of bits isn't an even 8. */
+        sizeB = (sizeB / 8) + ((sizeB % 8) ? 1 : 0);
+        
+        ptrA = (uint8_t*)XMALLOC(LTC_MAX_INT_BYTES, NULL, DYNAMIC_TYPE_BIGINT);
+        ptrB = (uint8_t*)XMALLOC(sizeB, NULL, DYNAMIC_TYPE_BIGINT);
+        if (ptrA == NULL || ptrB == NULL) {
+            res = MEMORY_E;
+        }
+
+    #ifndef WC_NO_RNG
+        if (res == MP_OKAY && rng != NULL) {
+            res = wc_RNG_GenerateBlock(rng, ptrB, sizeB);
+        }
+    #else
+        res = NOT_COMPILED_IN;
```

Comment:
Looks like this would override MEMORY_E stored in ret above if WC_NO_RNG is defined.

Reply:
Fixed